src/box/field_def.c

```diff
@@ -120,6 +120,11 @@ const char *on_conflict_action_strs[] = {
 	/* [ON_CONFLICT_ACTION_DEFAULT]  = */ "default"
 };
 
+const char *compression_type_strs[] = {
+	/* [COMPRESSION_TYPE_NONE]       = */ "none",
+	/* [COMPRESSION_TYPE_ZSTD]      = */  "zstd",
```

Comment:
We can remove this line under the macro, available only in the enterprise version. Then the test will need to be moved there too

Reply:
This may be better from the point of view of logic (why specify the type of compression if it is not supported?), on the other hand, we will get untested code in the free version

Reply:
We agreed to define enum and the string array differently for EE and OS versions.